### PR TITLE
Update eventing broker README

### PR DESCRIPTION
**Pull request by xtreme-sameer-vohra**
## Proposed Changes 
Add RabbitMQ as an example of a channel provider

### CODE REVIEW

**Comment by xtreme-sameer-vohra** [comment]
@googlebot I fixed it.

File: docs/eventing/broker/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Knative provides a multi-tenant, channel-based broker implementation that uses channels for event routing.
 
-Before you can use the Knative Channel-based Broker, you must install a channel provider, such as InMemoryChannel, Kafka or Nats.
+Before you can use the Knative Channel-based Broker, you must install a channel provider, such as InMemoryChannel, RabbitMQ, Kafka or Nats.
```
Comment (abrennan89) [inline docs/eventing/broker/README.md:3]:
```suggestion
Before you can use the Knative channel-based broker, you must install a channel provider, such as InMemoryChannel, RabbitMQ, Kafka or Nats.
```

**Review by abrennan89 — CHANGES_REQUESTED** [review]
If we're going to mention RabbitMQ, it should probably also be added here: https://knative.dev/docs/eventing/channels/channels-crds/ as an entry in the table.

We should try to keep this table in sync with the channel implementations we mention in this line probably, although I'm not sure we want to mention every single possible implementation here for "such as" - maybe just the core implementations, and add RabbitMQ to the table only?

WDYT @knative/eventing-wg-leads ?

**Comment by vaikas** [comment]
Actually, there's no RabbitMQ channel yet, and as such you can't install a RabbitMQ channel and use it with the channel based broker. It should be a separate broker implementation like this here:
Section 4 in the installing eventing component:
```Install a Broker (eventing) layer:```

just like there's Kafka broker, we should have a RabbitMQ broker as a separate tab.

My most sincere apologies for not catching this sooner, I just added the ok-to-test and was going to go look at it, and totally spaced, I'm sorry :(

**Comment by abrennan89** [comment]
Ahhh I see @vaikas - that makes sense!
Don't worry about it at all, it's no problem :slightly_smiling_face: it didn't merge or anything yet anyway.

@xtreme-sameer-vohra I am going to close this PR though without merging since this isn't accurate, hope that's OK.

**Comment by xtreme-sameer-vohra** [comment]
Thanks for the clarification. 
I hadn't looked further to check if RabbitMQ had a Channel implementation 🤦

**Pull request closed without merging**